README.md

```diff
@@ -1,18 +1,26 @@
 # BigImageViewer
 
-[ ![Download](https://api.bintray.com/packages/piasy/maven/BigImageViewer/images/download.svg) ](https://bintray.com/piasy/maven/BigImageViewer/_latestVersion)
+[![Download](https://api.bintray.com/packages/piasy/maven/BigImageViewer/images/download.svg)](https://bintray.com/piasy/maven/BigImageViewer/_latestVersion)
```

Comment:
The `<br />` is to provide some spacing between the gif and getting started section to make it easier to read but can remove it if you like.